src/main/java/yanagishima/util/QueryIdUtil.java

```diff
@@ -10,11 +11,16 @@
 
 @UtilityClass
 public final class QueryIdUtil {
+  private static final String FORMAT = "yyyyMMdd_HHmmss";
+
   public static String generate(String datasource, String query, String engine) {
-    String yyyyMMddHHmmss = ZonedDateTime.now(ZoneId.of("GMT")).format(
-        DateTimeFormatter.ofPattern("yyyyMMdd_HHmmss"));
+    String yyyyMMddHHmmss = ZonedDateTime.now(ZoneId.of("GMT")).format(DateTimeFormatter.ofPattern(FORMAT));
     double rand = Math.floor(Math.random() * 10000);
     return yyyyMMddHHmmss + "_" + DigestUtils.md5Hex(
         datasource + ";" + query + ";" + engine + ";" + ZonedDateTime.now() + ";" + rand);
   }
+
+  public static LocalDateTime datetimeOf(String queryId) {
+    return LocalDateTime.parse(queryId.substring(0, FORMAT.length()), DateTimeFormatter.ofPattern(FORMAT));
+  }
```

Comment:
Could you please add test in `QueryIdUtilTest` ?
For example,
```
assertEquals(LocalDateTime.of(2021, 5, 19, 0, 0, 0), QueryIdUtil.datetimeOf("20210519_000000_00000_aaaaa"));
```

Reply:
Added. However, I don't think such tests for simple utility method are helpful.